chapters/pipelines.adoc

```diff
@@ -7838,10 +7838,10 @@ with identical pipeline identifiers in the same pipeline cache.
 
 Unlike most structures declared by the Vulkan API, all fields of this
 structure are written with the least significant byte first, regardless of
-host byte-order.
+host byte-order, and the headerVersion field is written as exactly 4 bytes.
```

Comment:
This is the SC-specific version of the structure (which should be updated as well). The vulkan version is at line 7793. 

And we suspect there might be other structures which need similar language.

Reply:
* VkDeviceFaultVendorBinaryHeaderVersionOneEXT the same language and should be updated as well.
* [VkValidationCacheHeaderVersionEXT](https://registry.khronos.org/vulkan/specs/latest/html/vkspec.html#VkValidationCacheHeaderVersionEXT) has a table which explicitly lists the offsets, so it might be ok. 
* VulkanSC also has VkPipelineCacheHeaderVersionSafetyCriticalOne structure and the validationVersion field needs the same handling. 

Feels like maybe we should make a common include for two paragraphs that are replicated numerous places. (Might need to parameterize it by the intended structure size).

Reply:
> Feels like maybe we should make a common include for two paragraphs that are replicated numerous places. (Might need to parameterize it by the intended structure size).

No need for a separate include - write it in one place then use asciidoc transclusion like we do for boilerplate in the limits and features structs.